DOCS-4776 ASM Serverless beta docs (#17019)
DataDog/documentation: content/en/serverless/securing_functions/_index.md

---
title: Securing Functions
kind: documentation
further_reading:
- link: "/security/application_security/"
  tag: "Documentation"
  text: "Application Security Management"
- link: "/security/application_security/how-appsec-works"
  tag: "Documentation"
  text: "How Application Security Works"
---

[Datadog Application Security Management (ASM)][2] provides observability into application-level attacks that aim to exploit code-level vulnerabilities, and into bad actors targeting your systems.

> ASM secures functions written in Python, Node, Go, Java, and .NET. Because ASM is built on top of Serverless APM, the setup steps are straightforward, just adding an environment variable.

Reviewer: ```suggestion
ASM secures functions written in Python, Node, Go, Java, and .NET. Because ASM is built on top of Serverless APM, you can set it up by adding an environment variable.
```

> ASM supports over 130 event rules across major threats such as injection attacks, cross-site scripting, security scanner, local file inclusion, and many more. 

Reviewer: ```suggestion
ASM supports over 130 event rules across major threats such as injection attacks, cross-site scripting, security scanner, local file inclusion, and more. 
```
